mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -732,18 +738,28 @@
         "MOM_Tip_RJ_7" "Tip: Having trouble hitting a tele-sync? Use +paint to mark your spawn location."
         "MOM_Tip_RJ_8" "Tip: Tapping your backwards key while airborne can help you slow down and gain more control." // yi yi
         "MOM_Tip_RJ_9" "Tip: Sometimes there is more than one way to beat a jump. Get creative!" // porkie
+        "MOM_Tip_RJ_10" "Tip: If center firing is disabled, when wallclimbing, you have to aim further left for your rockets to hit walls at your center." // Paypur
 
         // Sticky Jump
         "MOM_Tip_SJ_1" "Tip: An uncharged stickybomb is fired at 900 units per second, regardless of the player’s speed."
         "MOM_Tip_SJ_2" "Tip: Jump and crouch before detonating a stickybomb to receive a much more powerful boost."
         "MOM_Tip_SJ_3" "Tip: Crouching without jumping before detonating a stickybomb will allow a more horizontal boost."
         "MOM_Tip_SJ_4" "Tip: Shooting stickies as early possible is essential to improving your course times." // hex
-
+        "MOM_Tip_SJ_5" "Tip: The Stickybomb Launcher fires on +attack end instead of on +attack start." // Hellbent
```

Comment:
```suggestion
        "MOM_Tip_SJ_5" "Tip: The Stickybomb Launcher fires on +attack end (mouse release) instead of on +attack start (mouse press)." // Hellbent
```

Reply:
"Bind start" isn't very intuitive

Reply:
Would like to avoid the use of console commands in the tips entirely as the goal is to make the majority of players (at least new ones) not need the console.

This tip specifically I feel is not useful for those that know what `+attack` is.

Reply:
I wonder if it'd be simpler to say
```
The Stickybomb Launcher fires when the attack button is released, instead of when it is pressed.
```
Or something like that to avoid any specific bind names and such. Could perhaps mention charging too.

Reply:
Yeah I like cabbage's approach, clean and simple.